MANUAL_REGRESSION.md

```diff
@@ -0,0 +1,207 @@
+    - user should persist on the same screen
+2. **STAGING ONLY:** Type invalid US number (+1 234 567 8912) and click send
+    - user should see `Something's gone wrong` error
+    - user should persist on the same screen
+
+##### 8. Check happy path flow on other desktop browsers
+(on Safari, Firefox, IE11 and Microsoft Edge browsers)
+
+Go through the flow looking for layout/usability inconsistencies between browsers:
+1. Select one of the documents
+    - everything should be displayed properly and layout should not be broken
+2. Upload document
+    - everything should be displayed properly and layout should not be broken
+3. Upload face photo
+    - everything should be displayed properly and layout should not be broken
+
+##### 9. Check happy path flow on mobile browsers
+(on Android Google Chrome and iOS Safari browsers)
+
+Go through the flow looking for layout/usability inconsistencies between browsers:
+1. Select one of the documents
+    - everything should be displayed properly and layout should not be broken
+2. Upload document
+    - everything should be displayed properly and layout should not be broken
+3. Upload face photo
+    - everything should be displayed properly and layout should not be broken
+
+##### 9. Check the camera is mirroring
+(on an iOS and Android device; a laptop with camera; desktop or laptop with a third-party USB camera)
+1. Go to the face step
+2. Move your face to the left
+    - Make sure your face also moves to the left on camera feed (like looking at a mirror)
+
+##### 9. Check that custom strings can be passed
+0. Go to latest JsFiddle
+1. Add the following options to the initialisation params:
+  ```javascript
+  language: {
+    locale: 'fr',
+    phrases: {'welcome.title': 'Ouvrez votre nouveau compte bancaire'}
+  }
+  ```
+2. Then the title on the welcome screen should be 'Ouvrez votre nouveau compte bancaire'
+
+##### 10. Overriding strings for a supported language
+0. Go to latest JsFiddle
+1. Add the following options to the initialisation params:
+  ```javascript
+  language: {
+    locale: 'es',
+    phrases: {'welcome.title': 'A custom string'}
+  }
+  ```
+2. Then the title on the welcome screen should be 'A custom string'
+3. All the other strings should be in Spanish
+
+##### 11. Overriding strings for a supported language on mobile
+0. Go to latest JsFiddle
+1. Add the following options to the initialisation params:
+  ```javascript
+  language: {
+    locale: 'es',
+    localePhrases: {'capture.passport.front.title': 'A custom string'}
+  }
+  ```
+2. Select passport on the document selector screen
+3. Choose the cross device flow and send an SMS to your mobile
+4. The SMS should be in Spanish
+5. When you open the link on your mobile device, the title on the cross device client should be 'A custom string'
+6. All the other strings should be in Spanish
+
+##### 12. Upload a document in PDF format
+(on Google Chrome, Firefox, Safari, IE11 and Microsoft Edge browsers)
```

Comment:
We have an automated test for this scenario. It runs on Chrome by default (also should be able to run on Firefox - but we don't do it on CI yet) so I think we could at least remove Chrome from this test

Reply:
Yes, I will remove Chrome then but I think we should still perform manual tests on other browsers.

Reply:
Yes test should stay for other browsers